### PR TITLE
Improve OS X build

### PR DESCRIPTION
**Pull request by alkurbatov**
* Fix OS X build under the latest sc2api.
* Add Linux & OS X build instructions.

### CODE REVIEW

**Comment by davechurchill** [comment]
Thank you again for all your hard work on this!

File: README.md [hunk 1]
```diff
@@ -61,9 +61,47 @@ If the bot crashes or does other nasty things, please bear with me while I make
...
+$ cmake ../
+$ make
+```
+
+* Install the SC2 API libraries and headers to your system
+```bash
+# Assumming that you are located in the 'build' directory
+# after finish of previous step
+$ cd ../
+$ sudo cp build/
```
Comment (vors) [inline README.md]: is this line correct? `cp` takes 2 args...

**Comment by vors** [comment]
@alkurbatov indeed, thank you for writing down the copy instruction! Can you, please, specify what commit from the s2client-api this was written against? It will likely diverge over time.

File: README.md (hunk 1 above)
Comment (alkurbatov) [reply README.md]: This is a typo :( There is no need to do anything with the 'build' directory itself.

**Comment by alkurbatov** [comment]
Ho @vors 
Do you mean that it should be specified in the README?

**Comment by vors** [comment]
Sure, in README would be a good place imo.

**Comment by alkurbatov** [comment]
See #28

**Comment by pimmen89** [comment]
I recently updated #28 to mention the API revision in the REAMDE.md

**Pull request merged**